docker-compose.yml

```diff
@@ -0,0 +1,107 @@
+---
+version: '3.7'
+services:
+
+  imsearch:
+    image: imsearch-cpu:latest
+    container_name: ${NAMESPACE}-cpu
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+    - 5000:5000
+    environment:
+    - MONGO_URI=mongodb://mongodb:27017/
+    - REDIS_URI=redis://redis:6379/0
+    - STORAGE_MODE=local
+    depends_on:
+    - redis
+    - mongodb  
+    links:
+    - mongodb:mongodb
+    - redis:redis
+    networks:
+    - internal
+    - web
+
+  mongodb:
+    image: webhippie/mongodb:latest
+    container_name: ${NAMESPACE}-mongodb
+    restart: always
+    ports:
+    - 27017:27017
+    - 28017:28017
+    - 27019:27019
+    volumes:
+    - mongo-data:/var/lib/mongodb/db
+    - mongo-backup:/var/lib/backup
+    networks:
+    - internal
+    environment:
+    - MONGODB_AUTH=${MONGODB_AUTH}
+    - MONGODB_AUTH_DB=${MONGODB_AUTH_DB}
+    - MONGODB_DATABASE=${MONGODB_DATABASE}
+    - MONGODB_DBPATH=${MONGODB_DBPATH}
+    - MONGODB_PASSWORD=${MONGODB_PASSWORD}
+    - MONGODB_REST=${MONGODB_REST}
+    - MONGODB_ROOT_PASSWORD=${MONGODB_ROOT_PASSWORD}
+    - MONGODB_ROOT_USERNAME=${MONGODB_ROOT_USERNAME}
+    - MONGODB_USERNAME=${MONGODB_USERNAME}
+    - MONGODB_VERBOSE=${MONGODB_VERBOSE}
+
+  mongoexpress:
```

Comment:
Why is mongoexpress even required? Can't we just use plain mongo installation?

Reply:
I can remove it, it was just for checking what is stored inside

Reply:
That's my second handle sorry if confused you are